larq_compute_engine/mlir/tests/bitpack-weights.mlir

```diff
@@ -2,11 +2,11 @@
 
 // CHECK-LABEL: @bitpack_bconv2d_filters
 func @bitpack_bconv2d_filters(%arg0: tensor<256x32x32x1xi32>, %arg1: tensor<16xf32>, %arg2: tensor<16xf32>, %arg3: none) -> tensor<256x30x30x16xf32> {
-  %cst = constant dense<1.0> : tensor<16x3x3x3xf32>
+  %cst = arith.constant dense<1.0> : tensor<16x3x3x3xf32>
```

Comment:
I renamed all `constant dense` into `arith.constant dense` inside the MLIR files and all `ConstantOp` into `Arith_ConstantOp` in the `*.td` files. This made most of the MLIR tests pass again, but it might have been too much: we should understand what the difference between the two is, since there are still a few places inside TensorFlow where `ConstantOp` is used in favor of `Arith_ConstantOp`.